src/models/vehicle/Vehicle.jsx

```diff
@@ -111,7 +115,7 @@ function VehicleAudio() {
     if (honk) {
       if (!honkAudio.current.isPlaying) honkAudio.current.play()
     } else honkAudio.current.isPlaying && honkAudio.current.stop()
-    if (brake && state.speed > 20) {
+    if (brake && state.speed > 20 && rightRearWheel.position.y < 0.29 && leftRearWheel.position.y < 0.29) {
```

Comment:
can the hardcoded values be changed to constants

Reply:
I meant the speed.. I don't think we are going to be using the wheel position like that since @drcmda commented about the `isInContact` prop:

> good idea, we need this! though cannon can tell you if wheels are in the air https://pmndrs.github.io/cannon-es/docs/classes/wheelinfo.html#isincontact it isnt exposed in use-cannon yet but i think that would be more flexible than testing position.y. i'll have a look tomorrow in the evening.

Reply:
@njm222 the speed is already a constant or you are talking about the 20 of number in the condition?

Reply:
Shweet! Don't forget to bump the version of `use-cannon`

Reply:
i dont think its yet possible, needs a quick fix so that nested props can be observed